dataactvalidator/filestreaming/fieldCleaner.py

```diff
@@ -120,6 +122,32 @@ def cleanLength(length):
             raise ValueError("Length must be positive")
         return length
 
+    @staticmethod
+    def cleanRow(row):
+        for key in row.keys():
+            value = row[key]
+            if value is not None:
+                tempValue = value.replace(",","")
+                if FieldCleaner.isNumeric(tempValue):
+                    row[key] = tempValue
+        return row
+
+    @staticmethod
+    def isNumeric(data):
+        try:
+            int(data)
+            return True
+        except:
+            try:
+                Decimal(data)
```

Comment:
Are there cases that would pass long but would not pass float?  We may need both of those.